### PR TITLE
Added Keynote Speaker Announcement for Alolita Sharma

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
CC: @palnabarun @sayanchowdhury @vipulgupta2048

### CODE REVIEW

File: content/blog/2020-announcing-keynote-speaker-alolita.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Title:Announcing Keynote Speaker for PyCon India 2020 - Alolita Sharma
+Date: 2020-09-08 17:00
+Author: Nikhil Maan (@Sc0rpi0n101)
+Category: 2020
+Slug: 2020-keynote-announcement-alolita-sharma
+Summary: We are happy to announce **Alolita Sharma** as a Keynote speaker for PyCon India 2020!
```
Comment (sayanchowdhury) [inline content/blog/2020-announcing-keynote-speaker-alolita.md:6]:
```suggestion
Summary: We are happy to announce **Alolita Sharma** as a Keynote Speaker for PyCon India 2020!
```

File: content/blog/2020-announcing-keynote-speaker-alolita.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+We are happy to announce **Alolita Sharma** as a Keynote speaker for PyCon India 2020!
```
Comment (sayanchowdhury) [inline content/blog/2020-announcing-keynote-speaker-alolita.md:8]:
```suggestion
We are happy to announce **Alolita Sharma** as a Keynote Speaker for PyCon India 2020!
```

**Review by sayanchowdhury — CHANGES_REQUESTED** [review]

File: content/blog/2020-announcing-keynote-speaker-alolita.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Title:Announcing Keynote Speaker for PyCon India 2020 - Alolita Sharma
+Date: 2020-09-08 17:00
+Author: Nikhil Maan (@Sc0rpi0n101)
+Category: 2020
+Slug: 2020-keynote-announcement-alolita-sharma
```
Comment (palnabarun) [inline content/blog/2020-announcing-keynote-speaker-alolita.md:5]:
```suggestion
Slug: keynote-announcement-alolita-sharma
```

**Review by palnabarun — CHANGES_REQUESTED** [review]
Just a small nit. 2020 is repeated in the URL due to it being in the article slug as well.

**Review by palnabarun — APPROVED** [review]

**Review by sayanchowdhury — APPROVED** [review]

File: content/blog/2020-announcing-keynote-speaker-alolita.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+Title:Announcing Keynote Speaker for PyCon India 2020 - Alolita Sharma
+Date: 2020-09-08 17:00
+Author: Nikhil Maan (@Sc0rpi0n101)
+Category: 2020
+Slug: keynote-announcement-alolita-sharma
+Summary: We are happy to announce **Alolita Sharma** as a Keynote Speaker for PyCon India 2020!
+
+We are happy to announce **Alolita Sharma** as a Keynote Speaker for PyCon India 2020!
+
+Alolita Sharma is a Principal Technologist at Amazon Web Services. She leads open source strategy and engineering for projects focused on observability and search including OpenTelmetry, Prometheus, Grafana, Apache Lucene and Open Distro for Elasticsearch. Two decades of doing open source continue to inspire her. Alolita has built and led engineering teams at Wikipedia, Twitter, PayPal and IBM. Alolita is passionate about language technologies that enable everyone to participate in the Web's information revolution. She serves on the board of [Unicode Consortium](https://www.unicode.org/consortium/consort.html) and is an invited expert on W3C and ECMA workgroups. She has served on the boards of the [Open Source Initiative (OSI)](https://opensource.org/) and [SFLC.in](https://sflc.in/).
```
Comment (palnabarun) [inline content/blog/2020-announcing-keynote-speaker-alolita.md:10]:
```suggestion
Alolita Sharma is a Principal Technologist at Amazon Web Services. She leads open-source strategy and engineering for projects focused on observability and search including OpenTelemetry, Prometheus, Grafana, Apache Lucene, and Open Distro for Elasticsearch. Two decades of doing open source continue to inspire her. Alolita has built and led engineering teams at Wikipedia, Twitter, PayPal, and IBM. Alolita is passionate about language technologies that enable everyone to participate in the Web's information revolution. She serves on the board of [Unicode Consortium](https://www.unicode.org/consortium/consort.html) and is an invited expert on W3C and ECMA workgroups. She has served on the boards of the [Open Source Initiative (OSI)](https://opensource.org/) and [SFLC.in](https://sflc.in/). 
```

**Review by palnabarun — CHANGES_REQUESTED** [review]

**Review by palnabarun — APPROVED** [review]

**Pull request merged**